lib/service.star

```diff
@@ -1,15 +1,42 @@
 def get_contract_setup_addresses(plan, args):
+    if args["zkevm_rollup_manager_address"]:
+        get_rollup_info_artifact = plan.get_files_artifact(
+            name="get-rollup-info-artifact",
+        )
+        result = plan.run_sh(
+            description="Retrieving rollup info",
+            image=args["toolbox_image"],
+            run="chmod +x {0} && sh {0} {1} {2} {3}".format(
+                "/opt/zkevm/get-rollup-info.sh",
+                args["l1_rpc_url"],
+                args["zkevm_rollup_manager_address"],
+                args["zkevm_rollup_chain_id"],
+            ),
+            files={"/opt/zkevm": get_rollup_info_artifact},
+        )
+
+        plan.print(result)
+        plan.print(result.output)
+        plan.print(
+            json.decode(
+                '{"zkevm_bridge_address": "0x1Fe038B54aeBf558638CA51C91bC8cCa06609e91"}'
+            )
+        )
+        contract_addresses = json.decode("\"" + result.output + "\"")
```

Comment:
```bash
INFO[2024-05-07T17:20:15+02:00] Executing Starlark package at '/Users/leovct/Documents/work/infra/kurtosis-cdk' as the passed argument '.' looks like a directory 
INFO[2024-05-07T17:20:15+02:00] Compressing package 'github.com/0xPolygon/kurtosis-cdk' at '.' for upload 
INFO[2024-05-07T17:20:23+02:00] Uploading and executing package 'github.com/0xPolygon/kurtosis-cdk' 
There was an error interpreting Starlark code 
Evaluation error: json.decode: at offset 2, unexpected character 'k'
	at [github.com/0xPolygon/kurtosis-cdk/main.star:200:44]: run
	at [github.com/0xPolygon/kurtosis-cdk/cdk_central_environment.star:61:53]: run
	at [github.com/0xPolygon/kurtosis-cdk/cdk_central_environment.star:103:76]: create_dac_config_artifact
	at [github.com/0xPolygon/kurtosis-cdk/lib/service.star:19:31]: get_contract_setup_addresses
	at [0:0]: json.decode

Error encountered running Starlark code.
```

Reply:
This way of retrieving data goes against Starlark's philosophy and future references. Another way is to deploy a helper service, retrieve the information using a simple script and extract the data from the file when the package is run.